### PR TITLE
fix(bw): add missing UA characters, mapping

### PR DESCRIPTION
**Pull request by 3djc**
This reworks the support for UA:
- only add necessary chars and use substitution for chars having a latin equivalent
- match all UA fonts
- automatically disables PXX2 for low flash radio so that UA can build
 
Thanks to the contributions of Salo, Безлактозний Мармелад on EdgeTX discord translation channel

Requires the awesome work by Philmoz in #4816

### CODE REVIEW

**Comment by raphaelcoeffic** [comment]
Should be rebased once #4816 has been merged.

**Comment by philmoz** [comment]
You might be able to save a few bytes of FLASH by changing the lookup tables in utf8.cpp from wchar_t to uint16_t.

File: radio/src/targets/taranis/CMakeLists.txt [hunk 1]
```diff
@@ -507,6 +507,10 @@ else()
   message( FATAL_ERROR "Unknown CPU_TYPE_FULL" )
 endif()
 
+if(TRANSLATIONS STREQUAL UA AND NOT TARGET_FLASH_SIZE STREQUAL 1M)
```
Comment (pfeerick) [inline radio/src/targets/taranis/CMakeLists.txt:510]:
```suggestion
if(NOT TARGET_FLASH_SIZE STREQUAL 1M AND (TRANSLATIONS STREQUAL RU OR TRANSLATIONS STREQUAL UA))
```

If you're going to do that, might as well do `RU` for completeness as it that suffers from the same issue. I tried building all languages for TX12MK2 (except for CN, JP, TW), and it was the only other one to fail.

I'll leave the rebase for you if you want to consider Phil's comment.

**Comment by 3djc** [comment]
> You might be able to save a few bytes of FLASH by changing the lookup tables in utf8.cpp from wchar_t to uint16_t.

Saves around 200 on a UA firmware :)

**Pull request merged**